resources/views/external-link-confirm.blade.php

```diff
@@ -6,61 +6,95 @@
 
 @php($identifier = md5($url . '-' . Str::random(8)))
 
-<a
-    href="javascript:;"
-    x-data="{
+<div x-data="{
         openModal() {
             Livewire.emit('openModal', '{{ $identifier }}')
+        },
+        redirect() {
+            window.open('{{ $url }}', '_blank')
+        },
+        hasDisabledLinkWarning() {
+            return localStorage.getItem('has_disabled_link_warning') === 'true';
         }
     }"
-    class="inline-flex items-center space-x-2 font-semibold whitespace-nowrap cursor-pointer link"
-    @click="openModal"
 >
-    <span>{{ $text ?? $slot ?? '' }}</span>
+    <a
+        :href="hasDisabledLinkWarning() ? '{{ $url }}' : 'javascript:;'"
+        :target="hasDisabledLinkWarning() ? '_blank' : '_self'"
```

Comment:
Sorry @Highjhacker, I was not clear! I mean adding here.
```suggestion
        :target="hasDisabledLinkWarning() ? '_blank' : '_self'"
        :rel="hasDisabledLinkWarning() ? 'noopener nofollow' : ''"
```

Reply:
My bad too on that one, updated :toot: